vpr/src/route/rr_graph_reader.cpp

```diff
@@ -72,8 +72,8 @@ void load_rr_file(const t_graph_type graph_type,
         device_ctx.rr_segments,
         device_ctx.physical_tile_types,
         grid,
-        &device_ctx.rr_node_metadata,
-        &device_ctx.rr_edge_metadata,
+        &device_ctx.rr_graph_builder.rr_node_metadata(),
```

Comment:
If the two APIs ``rr_node_metadata()`` and ``rr_edge_metadata()`` already return reference, you may not need the ``&`` here. Please double check.

Reply:
I tried to remove the & operator here and it starts throwing error of "no known conversion for arguments" we have to make changes to the locations where this function is used. So should I go for the changings or keep it in the way as it is?

Reply:
It is o.k. here. We may consider to change it later.